src/libloader/CMakeLists.txt

```diff
@@ -5,5 +5,27 @@ add_sources (elektra-full "static.c")
 if (BUILD_SHARED)
 	add_sources (elektra-shared "dl.c")
 	add_includes (elektra-shared)
+
+	if (${CMAKE_SYSTEM_NAME} STREQUAL "Windows")
+		set(BUILD_ARGS -DCMAKE_PREFIX_PATH=${CMAKE_PREFIX_PATH}
+			-DCMAKE_INSTALL_PREFIX:PATH=<INSTALL_DIR>
+			-DCMAKE_C_FLAGS=${CMAKE_C_FLAGS}
```

Comment:
there are no previous build args and I don't want to pass libelektra specific build arguments to dlfcn-win32